operators/src/engine/execution_context.rs

```diff
@@ -131,6 +131,7 @@ impl MockExecutionContext {
         MockQueryContext {
             chunk_byte_size,
             thread_pool: self.thread_pool.clone(),
+            extensions: Default::default(),
             abort_registration,
             abort_trigger: Some(abort_trigger),
         }
```

Comment:
I copied the session token from the UI to swagger-UI and /session works.

But /quota gives the following error:

![grafik](https://user-images.githubusercontent.com/4947061/199683564-d8818644-3134-47dc-9c6a-eada480040f9.png)

Issuing this directly works:

```
curl -X 'GET' \
  'http://127.0.0.1:3030/quota' \
  -H 'accept: application/json' \
  -H 'Authorization: Bearer 8ef8f6e9-f465-4eba-b743-ff732833dfe5'
``

Reply:
is there anything printed in the log of geo engine?

Reply:
![grafik](https://user-images.githubusercontent.com/4947061/199949991-9c452c72-a3d1-4e61-9a9f-3c59ff834355.png)

The backend seems to get the request
> 2022-11-04T10:20:21.380003Z  INFO actix_web::middleware::logger: 127.0.0.1 "POST /anonymous HTTP/1.1" 200 319 "http://localhost:3030/" "Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:106.0) Gecko/20100101 Firefox/106.0" 0.000434

Reply:
https://trello.com/c/OesNSvEg/1080-cors-f%C3%BCr-swagger-ui-in-der-entwicklung-fixen

Reply:
ok yes it may be a problem then with 127.0.0.1 vs localhost.